src/main/webapp/WEB-INF/view/login.jsp

```diff
@@ -31,6 +31,7 @@
       <a href="/login">Login</a>
     <% } %>
     <a href="/about.jsp">About</a>
+    <a href="/profile">Profile</a>
```

Comment:
Same as prev comment, order of pages in header

Reply:
fixed